lib/modules/manager/cnb/readme.md

```diff
@@ -0,0 +1,25 @@
+The `cnb` manager updates Cloud Native Buildpacks project descriptors in `project.toml` files.
```

Comment:
@rarkins do you want to keep calling it `cnb`, or do you want to rename to `buildpack`?

Context:

> > I'm pretty sure the CNF now have a trademark in buildpacks so this will be the only valid usage of that name
> 
> I don't think that matters since the v2 buildpacks are in CNCF, aren't they?
> 
> But probably the name doesn't really matter anyway. The end user will not have any contact unless a `project.toml` is present and for internal use there is still the `readme.md` if in doubt.

Users will still see the `cnb` name in their Renovate config. To me it seems easier to have a readable name like `buildpack`, than an abbreviation `cnb`.

Reply:
I would like it called `buildpacks` with an "s" at the end

Reply:
the manager has been renamed